recipes/websocketpp/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "websocketpp" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/zaphoyd/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: 6ce889d85ecdc2d8fa07408d6787e7352510750daa66b5ad44aacb47bea76755
+    patches:
+      - cmake-configversion-compatibility-anynewerversion.patch
+      - fix-cmake-find-boost.patch
+      - fix-cmake-install-path.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('websocketpp', max_pin='x.x.x') }}
```

Comment:
Why do we need a run exports if this is a header only library?

Reply:
Very true, thanks for the catch!